src/olympia/activity/views.py

```diff
@@ -37,3 +37,17 @@ def check_object_permissions(self, request, obj):
         # Just loading the add-on object triggers permission checks, because
         # the implementation in AddonChildMixin calls AddonViewSet.get_object()
         self.get_addon_object()
+
+    def get_serializer_context(self):
+        ctx = super(VersionReviewNotesViewSet, self).get_serializer_context()
+        ctx['to_highlight'] = self.pending_queryset(
+            amo.LOG.DEVELOPER_REPLY_VERSION)
+        return ctx
+
+    def pending_queryset(self, log_type):
+        version_qs = self.get_queryset()
+        dev_replies = version_qs.filter(action=log_type.id)
+        if len(dev_replies) == 0:
+            return version_qs
+        latest_reply = dev_replies[0]
```

Comment:
This looks inefficient, especially ``len(dev_replies)`` evaluates the query and fetches all entries. So, if we ever only use the first one we could write this as

```python
latest_dev_reply = version_qs.filter(action=log_type.id).first()
if not latest_dev_reply:
    return version_qs
return version_qs.filter(created__gt=latest_dev_reply.created)
```